Tools/LyTestTools/tests/integ/sanity_tests.py

```diff
@@ -15,40 +15,51 @@
 import ly_test_tools.environment.process_utils as process_utils
 import ly_test_tools.environment.waiter as waiter
 
-pytestmark = pytest.mark.SUITE_smoke
-
 logger = logging.getLogger(__name__)
 
 # Note: For device testing, device ids must exist in ~/ly_test_tools/devices.ini, see README.txt for more info.
 
 
 @pytest.mark.parametrize("project", ["AutomatedTesting"])
 class TestAutomatedTestingProject(object):
+
     def test_StartGameLauncher_Sanity(self, project):
+        # Kill processes that may interfere with the test
```

Comment:
Comments are definitely improved, but I think more of the guide should live inline in this code, per suggestion in https://github.com/o3de/o3de.org/pull/709

Reply:
added guide to inline comments